src/bin/dbpedia/getDBpediaOntology.sparql

```diff
@@ -0,0 +1,17 @@
+PREFIX dbo: <http://dbpedia.org/ontology/>
```

Comment:
I'd rename to `src/bin/dbpedia/getOntology.sparql` to keep consistent with the script